### PR TITLE
Add example service account etc for example.

### PR DESCRIPTION
**Pull request by mejlholm**
I ended up spending some time getting access to the API working, so i thought it would be nice to share the solution. 

If the example is too much, perhaps just include some links to get people going faster.

### CODE REVIEW

**Comment by gsmet** [comment]
@gastaldi @geoand could you have a look.

@mejlholm could you squash the commits a bit? 5 commits are a bit too much for this change, 2 would be better. Thanks!

**Comment by geoand** [comment]
Looks good to me!

Let's squash however as @gsmet said

File: docs/src/main/asciidoc/kubernetes-client.adoc [hunk 1]
```diff
@@ -220,3 +220,47 @@ public class ResourceWatcher<T extends HasMetadata> implements Watcher<T> {
 client.pods().watch(new ResourceWatcher<Pod>());
 ----
 
+== Access to the Kubernetes API
+
+You will most likely need access to the kubernetes API using a ServiceAccount, Role and RoleBinding to run the new application.
```
Comment (geoand) [inline docs/src/main/asciidoc/kubernetes-client.adoc]: Perhaps we should say: `In many cases in order to access the Kubernetes API server a ServiceAccount, Role and RoleBinding will be necessary`

File: docs/src/main/asciidoc/kubernetes-client.adoc [hunk 2]
```diff
@@ -220,3 +220,47 @@ public class ResourceWatcher<T extends HasMetadata> implements Watcher<T> {
...
+An example would look something like this:
```
Comment (geoand) [inline docs/src/main/asciidoc/kubernetes-client.adoc]: `An example that allows listing all pods could look something like this`

**Comment by mejlholm** [comment]
I'll just close this pull request, re-fork and request again.

**Pull request closed without merging**